sandbox/handlers/user_details.js

```diff
@@ -16,15 +16,15 @@ export async function user_details(req, res, next) {
   }
 
   if (!req.query || !req.query['ods-organisation-code']) {
-    sendError(res, 400, 'ods-organisation-code not provided.')
+    sendError(res, 400, 'you must specify an ODS code of non-zero length')
     next()
     return;
   }
 
   const odsCode = req.query['ods-organisation-code'].toUpperCase()
 
   if (!odsCodeRegex.test(odsCode) && odsCode !== 'X26') {
-    sendError(res, 400, 'Invalid ods-organisation-code value.')
+    sendError(res, 404, 'Not Found')
```

Comment:
Sorry this has probably been discussed but why is the mock returning a 404 instead of 400 for an invalid ODS code? That's not how the actual API behaves unless I am mistaken.

Reply:
When I tested, against my backend environment, I got a 404, 'Not found', so I changed the sandbox to be consistent with what I saw.

Reply:
Maybe our BE translates the 400 we get from their API to a 404. It seems strange to me but there might be a good reason for it.

Reply:
Did this get resolved? We shouldn't be translating 400s to 404s